apps/docs/components/navigation/footer.tsx

```diff
@@ -1,91 +1,93 @@
 import { PrivacySettingsLink } from '@/app/analytics'
 import { Logo } from '@/components/common/logo'
 import { TldrawLink } from '../common/tldraw-link'
+import { SocialLink } from './social-link'
 
 const menus = [
-	{
-		heading: 'Company',
-		items: [{ caption: 'Jobs', href: '/jobs' }],
-	},
 	{
 		heading: 'Product',
 		items: [
-			{ caption: 'Overview', href: '/' },
-			{ caption: 'Playground', href: 'https://tldraw.com' },
-			{ caption: 'Features', href: '/#features' },
-			{ caption: 'Pricing', href: '/#pricing' },
-			{ caption: 'FAQ', href: '/#faq' },
+			{ caption: 'Whiteboard', href: '/features/out-of-the-box-whiteboard' },
+			{ caption: 'Starter kits', href: '/starter-kits' },
+			{ caption: 'Pricing', href: '/pricing' },
+			{ caption: 'FAQ', href: '/faq' },
 		],
 	},
 	{
-		heading: 'Documentation',
+		heading: 'Developers',
 		items: [
-			{ caption: 'Learn', href: '/quick-start' },
-			{ caption: 'Reference', href: '/reference/editor/Editor' },
-			{ caption: 'Examples', href: '/examples' },
+			{ caption: 'Quick start guide', href: '/quick-start' },
+			{ caption: 'Starter kits', href: '/starter-kits' },
+			{ caption: 'Examples', href: '/examples/basic' },
+			{ caption: 'Releases', href: '/releases-versioning' },
```

Comment:
```suggestion
			{ caption: 'Releases', href: '/releases' },
```

We immediately redirect to `/releases` so maybe we just direct there immediately.